estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
```suggestion
from . import models

```
Add a new line at the end of each file

Reply:
noted , I will add new empty lines at end of code.